src/routes/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { BrowserRouter as Router, Route } from 'react-router-dom'
+
+import Auctions from 'components/Test_Auctions'
+import SellOrders from 'components/Test_SellOrders'
```

Comment:
Would it make sense to separate pages and components?

Reply:
this is testing stuff, for now this way is simpler
when we actually have routes, we'll revisit this point

Reply:
Ok...

I just thought it's as difficult, and I understood this PR's goal to be defining a project structure and stack to start with (since functionally we are not set yet).

I don't mind to do it later if you prefer it like that